src/server/components/dashboard/listsItems/item/update/actionHandlers/requestChanges.ts

```diff
@@ -0,0 +1,108 @@
+import { ListItemGetObject, User } from "server/models/types";
+import { logger } from "server/services/logger";
+import { initialiseFormRunnerSession } from "server/components/formRunner/helpers";
+import { getListItemContactInformation } from "server/models/listItem/providers/helpers";
+import serviceName from "server/utils/service-name";
+import { sendEditDetailsEmail } from "server/services/govuk-notify";
+import { AuditEvent, ListItem, Status } from "@prisma/client";
+import { prisma } from "server/models/db/prisma-client";
+import { EVENTS } from "server/models/listItem/listItemEvent";
+import { recordListItemEvent } from "server/models/audit";
+import { ListItemRes } from "server/components/dashboard/listsItems/types";
+import { Request } from "express";
+
+/**
+ * TODO: remove underTest
+ */
+export async function requestChanges(req: Request, res: ListItemRes) {
+  const { underTest } = req.params;
+  const isUnderTest = underTest === "true";
+  const userId = req.user!.id;
+  const changeMessage = req.session.update?.message;
+  const { listItem } = res.locals;
+  const { listItemUrl, listIndexUrl } = res.locals;
+
+  if (!listItem) {
+    logger.error("List item could not be found during requestChanges");
+    return res.status(500);
```

Comment:
yep, mb

Reply:
I've removed this now. should be handled by app.params. if ListItem is undefined and is accessed, like ListItem.jsonData, it will throw an error and be caught by the generic error handler.